.travis.yml

```diff
@@ -5,20 +5,20 @@ dist: trusty
 node_js:
   - "lts/*"
 
-env:
-  - TEST_DIR=cothority
-  - TEST_DIR=kyber
+# env:
+#   - TEST_DIR=cothority
+#   - TEST_DIR=kyber
 
-install:
-  - gimme 1.11
-  - . $HOME/.gimme/envs/go1.11.env
-  - go get github.com/dedis/Coding || true
-  # Because we are using "language: node_js" the "git clone" is not in the
-  # GOPATH. So make a copy of it over where it is supposed to be.
-  - git clone . `go env GOPATH`/src/github.com/dedis/cothority
-  - (cd `go env GOPATH`/src/github.com/dedis/cothority && go get -t ./... )
+# install:
+#   - gimme 1.11
+#   - . $HOME/.gimme/envs/go1.11.env
+#   - go get github.com/dedis/Coding || true
+#   # Because we are using "language: node_js" the "git clone" is not in the
+#   # GOPATH. So make a copy of it over where it is supposed to be.
+#   - git clone . `go env GOPATH`/src/github.com/dedis/cothority
+#   - (cd `go env GOPATH`/src/github.com/dedis/cothority && go get -t ./... )
```

Comment:
why is this part commented out?

Reply:
Mostly to keep the "hack" to use the correct repos in the before_install to have everything at the same place